### ISSUE TITLE
chore(android): build warnings

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#6334]
```
[15:27:27][Step 5/6] getMappingFile(): Could not find method getMappingFileProvider() for arguments [] on object of type com.android.build.gradle.internal.api.ApplicationVariantImpl.
[15:27:27][Step 5/6] getMappingFile(): Could not find method getMappingFileProvider() for arguments [] on object of type com.android.build.gradle.internal.api.ApplicationVariantImpl.
[15:27:27][Step 5/6] WARNING: API 'variantOutput.getPackageApplication()' is obsolete and has been replaced with 'variant.getPackageApplicationProvider()'.
[15:27:27][Step 5/6] It will be removed at the end of 2019.
[15:27:27][Step 5/6] For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
[15:27:27][Step 5/6] To determine what is calling variantOutput.getPackageApplication(), use -Pandroid.debug.obsoleteApi=true on the command line to display more information.
```

https://build.palaso.org/viewLog.html?buildId=309466&buildTypeId=KeymanAndroid_Build&tab=buildLog&_focus=1522

**Comment by darcywong00** [comment]
Noting this is for stable-14.0. Likely because of old build dependencies

**Comment by darcywong00** [comment]
Fixed by #6224
